src/main/java/com/commercetools/sync/categories/CategorySync.java

```diff
@@ -294,13 +294,13 @@ private void prepareDraftsForProcessing(@Nonnull final List<CategoryDraft> categ
      * @param categoryDraft the category draft to check whether it's parent is missing or not.
      * @param keyToIdCache  the cache containing mapping of all existing category keys to ids.
      * @return the same identical supplied category draft. However, with a null parent field, if the parent is missing.
-     * @throws ReferenceResolutionException thrown if the parent key is not valid, either not set, reference not
-     *                                      expanded or UUID is used in place of key when it shouldn't.
+     * @throws ReferenceResolutionException thrown if the parent key is not either not valid, set or when reference is
```

Comment:
is not either not valid -> is either not valid

Reply:
thanks for the catch @ahmetoz 😊: a568063

Reply:
It was actually a bit wrong, fixed now: 82a1e55